### PR TITLE
create / port missing configs for srze

### PR DESCRIPTION
**Pull request by Whennya**
Adds configs for:
- ze_avalanche_reboot_beta7
- ze_conflict_v4_s2
- ze_ffxii_ridorana_cataract_b6
- ze_otakuroom_v5_6fs
- ze_serpentis_temple_v2a
- ze_2049_css1
- ze_ffxii_mt_bur_omisace_v6_beta6fix
- ze_randomizer_c1s4
- ze_terraria_v3_2
- ze_luciddreams_v3_6_f2
- ze_light_shadow_l1_4
- ze_immortal_flame_v1_2
- ze_frozentemple_b8_2
- ze_flower_a1_s4
- ze_system_v1
- ze_minecraft_universe_v1_5s
- ze_minecraft_journey_v1_test_2
- ze_minecraft_adventure_v1_2c
- ze_666_crazy_escape_v2_css2f
- ze_treecolada_v1_3
- ze_ut2004_convoy_v2_2_1
- ze_xenovation_v3_5_s
- ze_uyuni_v1_6_s2
- ze_the_last_chapter_v1_5_css
- ze_thousand_steps_b7c
- ze_three_eyes_v0_7s_fix
- ze_tibia_v4_1s3
- ze_timesink_css1
- ze_tkara_v1
- ze_totemo_roka_b5s
- ze_trials_v1fs
- ze_tyranny2_v1fix
- ze_valkvang_s1
- ze_aztecorum_b3
- ze_bathroom_v2_5s
- ze_binary_z4_s1
- ze_biohazard2_rpd_v4e_004
- ze_biohazard2_sewer_v4c
- ze_bioharzard3_nemesis_b5_2s
- ze_8bit_v5b.cfg
- ze_abandoned_project_v1_2.cfg
- ze_azathoth_v1_css.cfg
- ze_surf_happy_2_newbie_village_css.cfg
- ze_surf_happy_b7s2.cfg
- ze_verdant_v1.cfg

Fixes
- ze_offliner_v2b.cfg mis-format on "behemoth-remix.mp3"	
- ze_m0w0m_b2 ( already on live server but not updated here )

Incomplete 
- ze_luigismanison_fix2

### CODE REVIEW

File: musicname/ze_ffxii_mt_bur_omisace_v6_beta6fix.cfg
Comment (notkoen) [inline musicname/ze_ffxii_mt_bur_omisace_v6_beta6fix.cfg:1]: Latest version is `ze_ffxii_mt_bur_omisace_v6fix` right now, which has different music.

File: musicname/ze_treecolada_v1_3.cfg [hunk 1]
```diff
@@ -0,0 +1,8 @@
+"music"
+{
+    "nto-every-wall-is-a-door.mp3"      "NTO - Every Wall Is A Door"
+    "rain of amethyst-camellia.mp3"     "Camellia - Rain of Amethyst"
+    "road 96 - on the road by robert parker.mp3"       "Robert Parker - On The Road"
+    "rupert holmes - escape.mp3"        "Rupert Holmes - Escape"
+//    "throat singing meme.mp3"       "file name"
```
Comment (notkoen) [inline musicname/ze_treecolada_v1_3.cfg:7]: Should just remove this line because it's not a song.

**Review by notkoen — COMMENTED** [review]
This is a nitpick, but I want the configs to have consistent formatting. Keep indentation at spaces and keep spacing between the file name and music name consistent (2 spaces? 8 spaces?).

It is my fault that I did not explicitly mention that file names need to be lowercase. I've added reviews on configs that had issues, and I'll wait until this gets merged before I update the README for other people.

Good work. I also want the other configs in the repository to be updated with consistent formatting, but that should be a separate PR.

File: musicname/ze_conflict_v4_s2.cfg [hunk 2]
```diff
@@ -0,0 +1,9 @@
+"music"
+{
+    "y2mate.com - northern lights.mp3"                                                  "Northen lights - 403"
+    "y2mate.com - nien - feel ma beat.mp3"                                              "FEEL MA BEAT - NieN"
+    "y2mate.com - deemo 3.1 - siromaru+cranky - conflict (vila remix).mp3"              "Deemo 3.1 - siromaru+cranky - Conflict (VILA Remix)"
+    "y2mate.com - cranky - old gold.mp3"                                                "Cranky - OLD GOLD"
+    "y2mate.com - [official] black gold - cranky _ tapsonic bold new song.mp3"          "[Official] BLACK GOLD - Cranky | TAPSONIC BOLD New song"
+    "y2mate.com - [mv] conflict  siromaru + cranky.mp3"                                 "[MV] conflict / siromaru + cranky"
```
Comment (notkoen) [inline musicname/ze_conflict_v4_s2.cfg]: Don't include extra details such as `[Official]` or `[MV]` in the song name.

File: musicname/ze_luciddreams_v3_6_f2.cfg [hunk 3]
```diff
@@ -0,0 +1,7 @@
+"music"
+{
+    "ahisa - Logical Stimulus.mp3"                          "a_hisa - Logical Stimulus"
+    "artcore avetadeltav & lyvia insensibilitv.mp3"         "AvEtaDeltaV & Lyvia - In_sensibilitv"
+    "artcore-veetacrush.mp3"                                "" // cant source cause source got taken down via dmca
```
Comment (notkoen) [inline musicname/ze_luciddreams_v3_6_f2.cfg]: Comment out missing song

File: musicname/ze_luigismanison_fix2.cfg [hunk 4]
```diff
@@ -0,0 +1,10 @@
+"music"
+{
+    "final musiclmz.mp3"        ""
+    "final_music_intro.mp3"     ""
+    "music1lmz.mp3"             "Bowser’s Castle - Mario Kart Double Dash"
+    "music2lmz.mp3"             "" // some bowsers castle ost, cant find specific
+    "music3lmz.mp3"             ""
+    "startmusiclmz.mp3"         ""
+  //"zombiedrunkbar.mp3"        ""
+}
```
Comment (notkoen) [inline musicname/ze_luigismanison_fix2.cfg]: Comment out missing songs

File: musicname/ze_m0w0m_b2.cfg [hunk 5]
```diff
@@ -1,28 +1,29 @@
...
+    "bgm_lvl5second.mp3"            "Monster Boy and the Cursed Kingdom - Volcano"
+    "bgm_lvl5boss.mp3"              "YouFulca - Legendary Opus"
+    "bgm_lvl5end.mp3"               "Super Smash Bros. for Wii U - Final Destination Ver. 2"
+    "bgm_lvl6first.mp3"             "Xenoblade Chronicles 2 - Counterattack"
+    "bgm_lvl6fourth.mp3"            "The End is Nigh OST - GLOOM Extended"
+    "bgm_lvl6second.mp3"            "Ridiculon - Udder Catastrophe - Big Slugger"
+    "bgm_lvl6third.mp3"             "xi - Ancient Relics"
+    "bgm_lvl6boss.mp3"              ""
+    "bgm_lvl6end.mp3"               "Cave Story - Last Battle remixed by BREIS"
 }
```
Comment (notkoen) [inline musicname/ze_m0w0m_b2.cfg]: Comment out missing songs

File: musicname/ze_offliner_v2b.cfg [hunk 6]
```diff
@@ -1,7 +1,7 @@
 "music"
 {
-	"siebzehn-sunset-on-mars.mp3"			"SIEBZEHN - Sunset On Mars"
+	"siebzehn-sunset-on-mars.mp3"			    "SIEBZEHN - Sunset On Mars"
 	"lazerhawk-arrival_offliner.mp3"			"Lazerhawk - Arrival"
-	"behemoth-remix.mp3"			"GosT - "Behemoth (Perturbator remix)"
-	"alex-tokyo-rose-sacrifice.mp3"			"Alex & Tokyo Rose - Sacrifice"
+	"behemoth-remix.mp3"			            "GosT - Behemoth (Perturbator remix)"
+	"alex-tokyo-rose-sacrifice.mp3"			    "Alex & Tokyo Rose - Sacrifice"
 }
```
Comment (notkoen) [inline musicname/ze_offliner_v2b.cfg]: Use spaces for formatting

File: musicname/ze_serpentis_temple_v2a.cfg [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+    "isaac.mp3"                                             "The Binding of Isaac OST - Main theme"
+    "matrix - brawl.mp3"                                    "Matrix reloaded - Burly Brawl"
+    "matrix - double trouble.mp3"                           "The Matrix Reloaded (OST) - Double trouble"
+    "matrix - kidfried.mp3"                                 "The Matrix Revolutions - Kidfried"
+    "matrix - neodammerung.mp3"                             "The Matrix Revolutions - Neodammerung"
+    "matrix - spirit of the universe.mp3"                   "The Matrix Revolutions - Spirit of the Universe"
+    "matrix - why mr anderson.mp3"                          "The Matrix Revolutions - Why Mr. Anderson?"
+    "mgs2 - theme.mp3"                                      "Metal Gear Solid 2 Soundtrack - Main Theme"
+    "mgs3 - theme.mp3"                                      "Metal Gear Solid 3 Soundtrack - Main Theme"
+    "run boy run - instrumental (woodkid).mp3"              "Woodkid - Run Boy Run Instrumental"
```
Comment (notkoen) [inline musicname/ze_serpentis_temple_v2a.cfg]: Nitpick: Should add `OST` for the other songs coming from Matrix, as well as keeping the title consistent (`Matrix reloaded` and `The Matrix Reloaded (OST)`?). Also, add parenthese to instrumental so it doesn't get confusing.

File: musicname/ze_system_v1.cfg [hunk 8]
```diff
@@ -0,0 +1,11 @@
+"music"
+{
+    "camelia - amorphous.mp3"                                               "Camelia - Amorphous"
+    "camellia - light it up (from crystallized).mp3"                        "Camelia - Light it Up"
+    "dancefloor hero.mp3"                                                   "K-Nu - Dancefloor Hero"
+    "gijensu - implosion (tim tama remix).mp3"                              "Gijensu - Implosion (Tim Tama Remix)"
+    "goreshit - burn this moment into the retina of my eye.mp3"             "Goreshit - Burn this moment into the retina of my eye"
+    "goreshit - fine night.mp3"                                             "Goreshit -  Fine Night"
```
Comment (notkoen) [inline musicname/ze_system_v1.cfg]: Extra space after `-`

File: musicname/ze_trials_v1fs.cfg [hunk 9]
```diff
@@ -0,0 +1,13 @@
...
+    "two_steps_from_hell_-_rise_above[mp3converter.net].mp3"                                "Two Steps From Hell - Rise Above"
+    "two_steps_from_hell_-_enchantress[mp3converter.net].mp3"                               "Two Steps From Hell - Enchantress"
+    "two steps from hell - stallion - epic music.mp3"                                       "Two Steps From Hell - Stallion"
+    "nier automata ost - song of the ancients - atonement (instrumental).mp3"               "NieR: Automata OST - Song of the Ancients ~ Atonement"
+    "nier automata ost - a beautiful song (instrumental).mp3"                               "NieR: Automata OST - A Beautiful Song (Instrumental)"
+    "level_1_boss_music.mp3"                                                                "Zero no Kiseki OST - Inevitable Struggle"
+    "ivan_torrent_-_icarus_feat_julie_elven[mp3converter.net].mp3"                          "Ivan Torrent (feat. Julie Elven) - Icarus"
+    "epic trailer music royalty free.mp3"                                                   "Grooving Gecko - Orcs are Coming"
+
+}
```
Comment (notkoen) [inline musicname/ze_trials_v1fs.cfg]: Remote whitespace

File: musicname/ze_uyuni_v1_6_s2.cfg [hunk 10]
```diff
@@ -0,0 +1,18 @@
+"music"
+{
+    "celestial_exploring.mp3"                       "a_hisa - Celestial Exploring"
+    "chase_the_light_a.mp3"                         "Adam Fielding - Chase The Light"
+    "chase_the_light_t.mp3"                         "Two Steps From Hell - Chase The Light (Amaria)"
+    "dream_of_the_shore_near_another_world.mp3"     "Chrono Cross OST - Dream of the Shore Near Another World"
+    "eternity.mp3"                                  "PIKASONIC - Eternity"
+    "liyuebattle.mp3"                               ""
```
Comment (notkoen) [inline musicname/ze_uyuni_v1_6_s2.cfg]: Trim whitespace and comment out missing songs

File: musicname/ze_666_crazy_escape_v2_css2f.cfg [hunk 11]
```diff
@@ -0,0 +1,15 @@
...
+    "nrk1_room2_music.mp3"          "Ragnarok Online OST - Stranger Aeons"
+    "nrk1_room3_music.mp3"          "Ragnarok Online OST - Curse'n Pain"
+    "nrk1_room4_music.mp3"          "Ragnarok Online OST - Indelible Scars"
+    "nrk1_room5_music.mp3"          "HiroyukiSawano - Warcry"
+    "nrk2_room1_music.mp3"          "Ragnarok Online OST - Through the tower"
+    "nrk2_room2_music.mp3"          "Ragnarok Online OST - Help Yourself"
+    "nrk2_room3_music.mp3"          "Ragnarok Online OST - Angelica"
+    "nrk2_room4_music.mp3"          "Ragnarok Online OST - Labryinth"
+    "nrk2_room5_music.mp3"          "Ragnarok Online OST - Dark Light"
+    "nrk2_room6_music.mp3"          "『Chaos Drifters』- SawanoHiroyuki[nZk] & Jean-Ken Johnny"
```
Comment (notkoen) [inline musicname/ze_666_crazy_escape_v2_css2f.cfg]: `nrk2_room6_music.mp3` doesn't follow the `<Artist> - <Song name>` format.

File: musicname/ze_arcana_heart_css.cfg
Comment (notkoen) [inline musicname/ze_arcana_heart_css.cfg:1]: Consistent formatting

Comment (Whennya) [reply musicname/ze_arcana_heart_css.cfg:1]: https://github.com/Whennya/Music-Names/tree/consistent-formatting
its done in another branch, ill add it to pr later after this one is done / merged

Comment (notkoen) [reply musicname/ze_arcana_heart_css.cfg:1]: Noted

**Pull request merged**